config/application.rb

```diff
@@ -11,6 +11,8 @@
 
 require 'view_component/engine'
 
+require 'sprockets/railtie'
```

Comment:
CSS and JS still working on the review app. No notices on app boot. Nothing in our `assets` folder anyway. I think this is probably fine. 🤷